### ISSUE TITLE
dx value issue

### ISSUE THREAD

**Issue by ThibautVanHoof** [pylbm/pylbm_ui#43]
we try to define a implosion test case with 400 cells in both directions corresponding to a space step=0.00075. But automatic correction if the the space step value do not allows for that value. 
Best is 0.0006993006993006993 leading to 430 cells.

**Comment by bgraille** [comment]
I @ThibautVanHoof ,
I am not sure I understand the issue. If you take dx = 0.00075, you have 401 points in each direction, that corresponds to 400 cells. And in the xdmf output, you have indeed 400 cells:
```
<?xml version="1.0" ?>
<!DOCTYPE Xdmf SYSTEM "Xdmf.dtd" []>
<Xdmf>
 <Domain>           
            
                <Grid Name="Structured Grid" GridType="Uniform">
                    <Topology TopologyType="2DRectMesh" NumberOfElements="400 400"/>
                    <Geometry GeometryType="VXVY">
                
                <DataItem Format="HDF" Dimensions="400">
                    mass_6666.h5:/x_0 
                </DataItem>
                
                <DataItem Format="HDF" Dimensions="400">
                    mass_6666.h5:/x_1 
                </DataItem>
                </Geometry>

                <Attribute Name="mass" AttributeType="Scalar" Center="Node">
                <DataItem Format="HDF" Dimensions="400 400">
                mass_6666.h5:/mass 
                </DataItem>
                </Attribute>
                </Grid>
</Domain>
</Xdmf>
```

**Comment by bgraille** [comment]
fix in 83e171c

**Comment by ThibautVanHoof** [comment]
Very nice, thanks a lot again @bgraille
